Stratis.Bitcoin/Features/MemoryPool/Fee/TxConfirmStats.cs

```diff
@@ -1,73 +1,102 @@
+using Microsoft.Extensions.Logging;
+using NBitcoin;
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
-using Microsoft.Extensions.Logging;
-using NBitcoin;
 
 namespace Stratis.Bitcoin.Features.MemoryPool.Fee
 {
+    /// <summary>
+    /// Transation confirmation statistics.
+    /// </summary>
     public class TxConfirmStats
     {
+        /// <summary>Logger for logging messages for this object.</summary>
```

Comment:
We used the term "Instance logger." in other classes. Feel free to use it if you want.